specification/v3.3/OSDM-online-api-v3.3.0-draft.yml

```diff
@@ -8431,7 +8431,9 @@ components:
         Fulfillment types. RCT2, RCCST and UIC_PDF are official UIC standards.
       type: string
       x-extensible-enum:
-        - 'ALLOCATOR_APP'
+        - 'ALLOCATOR_APP' # deprecated
+        - 'DISTRIBUTOR-APP' 
```

Comment:
@CGantert345 Do we have a code style on code list values? Usually, underscore char is used.